### PR TITLE
fix: PIN validation passing with incorrect PIN

### PR DESCRIPTION
**Pull request by jp0707**
# Summary of Changes

Fix PIN validation passing with incorrect PIN.

# Related Issues

https://github.com/hyperledger/aries-mobile-agent-react-native/issues/912

# Pull Request Checklist

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

### CODE REVIEW

File: packages/legacy/core/App/components/inputs/PINInput.tsx [hunk 1]
```diff
@@ -23,13 +23,17 @@ const PINInput: React.FC<PINInputProps & React.RefAttributes<TextInput>> = forwa
...
     const [PIN, setPIN] = useState('')
     const [showPIN, setShowPIN] = useState(false)
-    const [props, getCellOnLayoutHandler] = useClearByFocusCell({
-      value: PIN,
-      setValue: setPIN,
-    })
     const { t } = useTranslation()
     const { TextTheme, PINInputTheme } = useTheme()
     const cellHeight = 48
+    const myOnChangeText = (value: string) => {
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/components/inputs/PINInput.tsx]: Looks good but can we name this function `onChangeText` and pass it directly to the `CodeField` prop of the same name?

File: packages/legacy/core/App/components/inputs/PINInput.tsx [hunk 2]
```diff
@@ -80,8 +84,7 @@ const PINInput: React.FC<PINInputProps & React.RefAttributes<TextInput>> = forwa
             value={PIN}
             rootStyle={style.codeFieldRoot}
             onChangeText={(value: string) => {
```
Comment (bryce-mcmath) [inline packages/legacy/core/App/components/inputs/PINInput.tsx]: As per above comment:

`onChangeText={onChangeText}`

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
One small change, otherwise good catch!

File: packages/legacy/core/App/components/inputs/PINInput.tsx (hunk 2 above)
Comment (jp0707) [reply packages/legacy/core/App/components/inputs/PINInput.tsx]: done

File: packages/legacy/core/App/components/inputs/PINInput.tsx (hunk 1 above)
Comment (jp0707) [reply packages/legacy/core/App/components/inputs/PINInput.tsx]: done

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**